extensions/schema-compare/src/schemaCompareResult.ts

```diff
@@ -13,6 +13,7 @@ import { Telemetry } from './telemetry';
 import { getTelemetryErrorType } from './utils';
 const localize = nls.loadMessageBundle();
 const diffEditorTitle = localize('schemaCompare.ObjectDefinitionsTitle', 'Object Definitions');
+const appliyConfirmation = localize('schemaCompare.ApplyConfirmation', 'Are you sure that you want to update the target?');
```

Comment:
nit: typo - appliyConfirmation

Reply:
Are you sure you want to apply changes to the target?

what is the message in SSDT?

Reply:
Yeah _that_ threw me off too but its the same msg in SSDT. Should I still remove _that_ ?
![image](https://user-images.githubusercontent.com/46980425/58515237-12438c80-8159-11e9-8267-72414d8b372b.png)

Reply:
@kisantia , native English speaker, please share your thoughts.

Reply:
Yes remove "that"